### PR TITLE
CB-17332 Adds the ability to use the latest CH in case of the VERSION is 'latest' or missing

### PR DESCRIPTION
**Pull request by pkedvessy**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,11 @@ endif
 
 GOFILES_NOVENDOR = $(shell find . -type f -name '*.go' -not -path "./vendor/*" -not -path "./.git/*")
 
+VERSION?=latest
+ifeq ($(VERSION),latest)
+VERSION=$(shell curl -s https://api.github.com/repos/hortonworks/cloud-haunter/releases/latest | jq -r '.name')
```
Comment (mhmxs) [inline Makefile:35]: This line would fail if someone doesn't have `jq`. I suggest solving this before merging.

Comment (Bajzathd) [reply Makefile:35]: +1. Also, it will run every time you do not specify a VERSION, so I think it should be moved under the download target.

Comment (pkedvessy) [reply Makefile]: Now it is jq free.

**Review by mhmxs — APPROVED** [review]
Could you please remove binary files. Thanks

**Comment by pkedvessy** [comment]
Sure, and I'm going to extend gitignore as well :-D

**Review by Bajzathd — APPROVED** [review]

**Pull request merged**